streamalert_cli/outputs/handler.py

```diff
@@ -71,32 +176,336 @@ def handler(cls, options, config):
             return False
 
         # get dictionary of OutputProperty items to be used for user prompting
-        props = output.get_user_defined_properties()
+        properties = output.get_user_defined_properties()
```

Comment:
Big fan of renaming `props` to `properties`. I'm a reactJS developer so `props` triggers me